source/onboard/migrating-to-mattermost.rst

```diff
@@ -100,78 +100,113 @@ When IT adopts management of Mattermost, often they will purchase the commercial
+
+.. note::
 
-4. Convert Slack import to Mattermost bulk export format
+  Make sure not to unzip and rezip the Slack export. Doing that can modify the directory structure of the archive which could cause issues with the import process.
+
+4. Convert Slack import to Mattermost bulk export format with ``mmetl``
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
-Now that you have a Slack export file with emails and attachments you have to convert it to the Mattermost format using ``mmetl``. Download the latest release of ``mmetl`` for your OS and architecture `here <https://github.com/mattermost/mmetl/releases/>`__ and extract it to your $PATH like with ``slack-advanced-exporter``. The same caveat applies.
+Now that you have a Slack export file with emails and attachments, let's convert this information into the "Mattermost bulk import" format using the import preparation tool ``mmetl``. Download the latest release of ``mmetl`` for your OS and architecture `here <https://github.com/mattermost/mmetl/releases/>`__ Run ``mmetl help`` to learn more about the tool.
 
-Next, run this command to do the conversion. Replace ``<TEAM NAME>`` with the name of your team:
+Next, run this command to create a Mattermost bulk import file. Replace ``<TEAM NAME>`` with the name of your team in Mattermost:
 
 .. code:: bash
 
     ./mmetl transform slack --team <TEAM NAME> --file export-with-emails-and-attachments.zip --output mattermost_import.jsonl
 
+Michael K Note:
+I don't understand the below pre-existing paragraph. I'm not sure what the zip file structure is after slack advanced export
+https://github.com/grundleborg/slack-advanced-exporter/blob/c0f7bf8230cd15d9b44af36d73ce8f0650538c67/cmd/fetch_attachments.go#L148
+It looks like we place them in a folder called ``__uploads``
+but it's my understanding that we should make a ``data`` folder. I'm not sure what ``bulk-export-attachments`` is representing below. I need to look into this more.
+I need to go through this part of the process. I generated a slack export, but it doesn't contain posts for "today", so I don't have the available info
+
 Next you have to create a zip file with the ``mattermost_import.jsonl`` file and the directory ``bulk-export-attachments`` (which needs to be moved to a subdirectory ``data`` first) that contains the attachments. On Linux and macOS you can use this command:
 
+
+The tool outputs a `.jsonl` file https://jsonlines.org/examples. We now need to place this file and attachments into a new zip file. Place this output file in the root directory of the zip file.
+
+Michael K note:
+it doesn't matter what th `.jsonl` file is called
+you can name it what you want with the `--output` flag as shown above
+it just needs to be a `.jsonl` file and be in the root directory of the zip file
+this is a point of confusion, since there is no clear understanding of what the name of the files needs to be
+
```

Comment:
Highlighting two existing comments here. The "Michael K Note" pieces will be removed. They are essentially TODOs right now